app/views/front/_menu.html.erb

```diff
@@ -7,10 +7,20 @@
             <% if !child_page.children.nil? %>
                 <ul>
                   <% child_page.children.each do |grandchild_page| -%>
-                      <li><a href="<%= grandchild_page.url %>"><%= grandchild_page.name %></a></li>
+                      <li>
+                        <a href="<%= grandchild_page.url %>"><%= grandchild_page.name %></a>
+                        <% if !grandchild_page.children.nil? %>
+                          <ul>
+                            <% grandchild_page.children.each do |child_page| -%>
```

Comment:
Can you replace the name of the variable `child_page` to something else? This variable already exist.